### PR TITLE
Splunkoncall: PAPP-23432 Added custom view for 'list oncalls' action

### PR DESCRIPTION
**Pull request by nmehra-crest**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: splunkoncall.json [hunk 1]
```diff
@@ -649,7 +649,7 @@
                 {
                     "data_path": "action_result.summary.entity_id",
                     "contains": [
-                        "victorops id"
+                        "splunk on call id"
```
Comment (alexa-phantom) [inline splunkoncall.json:652]: How does changing the contains affect contextual actions? ie could it make sense to keep both `victorops id` in addition to `splunk on call id`?

File: splunkoncall_connector.py [hunk 2]
```diff
@@ -401,7 +402,7 @@ def _handle_update_incident(self, param):
             "entity_display_name": param_name,
             "state_message": param_message
         }
-
+        self.debug_print("Endpoint created for _handle_create_update_incident action: {0}".format(endpoint))
```
Comment (alexa-phantom) [inline splunkoncall_connector.py:405]: Do we really need all the debug messages here and in rest of the code?

**Review by alexa-phantom — CHANGES_REQUESTED** [review]

File: splunkoncall_connector.py (hunk 2 above)
Comment (nmehra-crest) [reply splunkoncall_connector.py:405]: There is an update in the static test pipeline and according to that, we should have at least 2 logs statement in the methods. That's why we have made these changes in the code.

File: splunkoncall.json (hunk 1 above)
Comment (nmehra-crest) [reply splunkoncall.json:652]: Actually, we have replaced the 'victorops' with 'splunk on call' in the entire code. We have also verified the contextual actions where we have made changes in contains and they are working fine.

File: splunkoncall.json [hunk 3]
```diff
@@ -15,28 +15,28 @@
...
+            "description": "API Key from the Splunk On-Call Portal",
             "data_type": "password",
             "required": true,
             "order": 1
         },
         "integration_url": {
-            "description": "Custom REST based Integration URL defined in VictorOps",
-            "verbose": "Populate this field if you plan to create, update, or resolve incidents. i.e https://alert.victorops.com/integrations/generic/20131114/alert/84d34810-7b78-4cb9-91a4-3425aeecce68",
+            "description": "Custom REST based Integration URL defined in Splunk On-Call",
+            "verbose": "Populate this field if you plan to create, update, or resolve incidents. i.e https://alert.victorops.com/integrations/generic/20131114/alert/84d34810-7b78-4cb9-91a4-3425aeecce68.",
```
Comment (ricardo-phantom) [inline splunkoncall.json:39]: are the URLs also changing from alert.victorops.com to alert.splunkoncall.com?

Comment (btavethiya-crest) [reply splunkoncall.json:39]: We have checked there is no change in the URL so we kept it as it is.

**Review by alexa-phantom — APPROVED** [review]

**Review by ishans-crest — APPROVED** [review]

**Pull request merged**